### ISSUE TITLE
Problem with inputs render 

### ISSUE THREAD

**Issue by SantiSarmiento** [crazycodeboy/react-native-splash-screen#624]
I follow the steps on the github repo but the problem stills.

My problem occurs after installing and configuring React Native Splash Screen in my application. I followed all the steps correctly, and when I launch the app, the image I chose is displayed. However, when navigating between screens, if the screen contains an Input tag (I use the NativeBase library, but I also tried with the React component and encountered the same error), the app crashes and displays the following error message:

![error](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/6173ea5c-ee91-4a2f-abd7-2136211948db)

build.gradle file:

![build gradle](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/3b21e206-331b-4a03-9a8a-9c9e33992bcb)

settings.gradle file:

![settings gradle](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/3acb2448-40a0-4967-968e-c2b28efb993e)

MainActivity.java file:

![MainActivity](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/fa457331-85e7-40e5-9dcf-62f2c50443b5)

launch_screen.xml file:

![launch screen](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/8fc03799-8a62-419c-98a7-e0a6695aef18)

proyect structure: 

![proyect structure](https://github.com/crazycodeboy/react-native-splash-screen/assets/73916781/2976f068-ae69-4352-a2e8-32e945bf997e)

"Can you see anything wrong? Any ideas or suggestions?

**Comment by herbertcordeiro** [comment]
same problem here =/

**Comment by kelokchan** [comment]
Try to use this `launch_screen.xml`

```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:orientation="vertical" android:layout_width="match_parent"
    android:layout_height="match_parent">
    <ImageView android:layout_width="match_parent" android:layout_height="match_parent" android:src="@drawable/screen" android:scaleType="centerCrop" />
</LinearLayout>
```

**Comment by herbertcordeiro** [comment]
> Try to use this `launch_screen.xml`
> 
> ```
> <?xml version="1.0" encoding="utf-8"?>
> <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
>     android:orientation="vertical" android:layout_width="match_parent"
>     android:layout_height="match_parent">
>     <ImageView android:layout_width="match_parent" android:layout_height="match_parent" android:src="@drawable/screen" android:scaleType="centerCrop" />
> </LinearLayout>
> ```

Still the same problem, just changes that: "Class not found LinearLayout" :(

**Comment by SantiSarmiento** [comment]
> Try to use this `launch_screen.xml`
> 
> ```
> <?xml version="1.0" encoding="utf-8"?>
> <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
>     android:orientation="vertical" android:layout_width="match_parent"
>     android:layout_height="match_parent">
>     <ImageView android:layout_width="match_parent" android:layout_height="match_parent" android:src="@drawable/screen" android:scaleType="centerCrop" />
> </LinearLayout>
> ```

Already try that: Class not found LinearLayout

**Comment by phvasconcellos84** [comment]
did i remake the installation, adding color.xml that i've forgot and putting android:windowIsTranslucent = true on styles.xml. Now all is fine.

**Comment by GaneshBorkar6** [comment]
@phvasconcellos84, I'm also facing the same issue while using TextInput with navigation, as you have a workaround to this issue, I tried your approach to set android:windowIsTranslucent to true on styles.xml as,

< item name="android:windowIsTranslucent" > true </ item>

But this is still not working!

So can you please provide the code changes here, so that we can make the same changes and try this? Or else paste all the splash screen-related changes such as changes in the Android manifest file, and styles.xml file.

**Comment by Digital-Human-BK** [comment]
Has anyone found a solution? I've just hit the same error... My settings are the same as those of the post creator.

**Comment by fershibli** [comment]
Yep, same error, changing to LinearLayout just changed the error:
![image](https://github.com/crazycodeboy/react-native-splash-screen/assets/10034666/0dd2b2bb-9425-4fbb-ab20-861ef13c0980)

**Comment by sunjaaa** [comment]
it seems to crash on screens that use TextInput
The following worked perfectly for me

*see links below
[https://stackoverflow.com/questions/76773115/relativelayout-error-from-textinput-and-launch-screen-file/76879105#76879105](https://stackoverflow.com/questions/76773115/relativelayout-error-from-textinput-and-launch-screen-file/76879105#76879105)

**Comment by fershibli** [comment]
> it seems to crash on screens that use TextInput The following worked perfectly for me
> 
> *see links below https://stackoverflow.com/questions/76773115/relativelayout-error-from-textinput-and-launch-screen-file/76879105#76879105

You're amazing! TYSM! It worked perfectly. But why were we using this line on `styles.xml`? Is this a diference on the setup for previous versions? This should be highlighted on `README.md`

**Comment by fikkatra** [comment]
In my case, it was due to the following line in `styles.xml`. Removing this line resolved the issue:

`<item name="android:editTextBackground">@layout/launch_screen</item>`

This line is mentioned in some old documentation and blogposts, but is not necessary (anymore). Just adding launch_screen.xml in the layout directory, works fine.
